### ISSUE TITLE
laplace_transform not taking into account coefficients of derivatives.

### ISSUE THREAD

**Issue by jensena** [sympy/sympy#23360]
When applying a Laplace transform to the derivative of an undefined function, coefficients in front of the function are vanishing.

For example:
```python
import sympy as sp

t, s, a = sp.symbols("t s a")
y = sp.Function("y")

print(sp.laplace_transform(a*y(t).diff(t), t, s))
```

returns `s*LaplaceTransform(y(t), t, s) - y(0)`,

instead of `a*s*LaplaceTransform(y(t), t, s) - a*y(0)`.

This problem also occurs for defined constants e.g. `2*y(t).diff(t)` and for higher derivatives.

Applying the transform to `a*y(t)` retains the constant however: `a*LaplaceTransform(y(t), t, s)`

Thanks!

**Comment by oscarbenjamin** [comment]
I think this is already fixed in master from #23308

**Comment by jensena** [comment]
> I think this is already fixed in master from #23308

Thank you- I only searched "laplace_transform" and somehow missed the previous thread. I've closed this duplicate concern.
